kcidb/monitor/test_output.py

```diff
@@ -4,11 +4,13 @@
 import unittest
 from kcidb_io import schema
 from kcidb import orm, db, oo
-from kcidb.monitor.output import NotificationMessage, Notification
+from kcidb.monitor.output import NotificationMessage, Notification,\
+    NOTIFICATION_MESSAGE_SUBJECT_MAX_LEN, NOTIFICATION_MESSAGE_BODY_MAX_LEN
+
 
 # Disable long line checking for JSON data
 # flake8: noqa
-# pylint: disable=line-too-long
+# pylint: disable=line-too-long : disable-msg=too-many-locals
```

Comment:
You can just use one variable each for `NotificationMessage`, `Notification`, and `EmailMessage` throughout the code, now that you have comments explaining what each setup/assertion block does, and reuse them between blocks. This way you'll likely not need to disable this warning. But please don't use the same variable to store both `NotificationMessage` and `EmailMessage` objects, like you do now, that could be confusing.